src/main/java/ch/wisv/areafiftylan/products/model/TicketType.java

```diff
@@ -17,46 +17,62 @@
 
 package ch.wisv.areafiftylan.products.model;
 
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
 import java.time.LocalDateTime;
+import java.util.Collection;
+import java.util.HashSet;
 
-public enum TicketType {
-    EARLY_FULL("Early Bird", 37.50F, 50, LocalDateTime.of(2016, 6, 3, 0, 0), true),
-    REGULAR_FULL("Regular", 40.00F, 0, LocalDateTime.of(2016, 5, 28, 23, 59), true),
-    LAST_MINUTE("Last Minute", 42.50F, 0, LocalDateTime.of(2016, 6, 5, 23, 59), true),
-    TEST("Test Ticket", 999.0F, 0, LocalDateTime.MAX, true),
-    FREE("Free", 0F, 0, LocalDateTime.MAX, false);
+@Entity
+@NoArgsConstructor
+public class TicketType {
+    //    EARLY_FULL("Early Bird", 37.50F, 50, LocalDateTime.of(2016, 6, 3, 0, 0), true),
+    //    REGULAR_FULL("Regular", 40.00F, 0, LocalDateTime.of(2016, 5, 28, 23, 59), true),
+    //    LAST_MINUTE("Last Minute", 42.50F, 0, LocalDateTime.of(2016, 6, 5, 23, 59), true),
+    //    TEST("Test Ticket", 999.0F, 0, LocalDateTime.MAX, true),
+    //    FREE("Free", 0F, 0, LocalDateTime.MAX, false);
 
-    private final float price;
-    private final int limit;
-    private final String text;
-    private final LocalDateTime deadline;
-    private final boolean buyable;
+    @Id
+    @GeneratedValue
+    @Getter
+    private Long id;
+    @Getter
+    @Setter
+    private String name;
+    @Getter
+    @Setter
+    private float price;
+    @Getter
+    @Setter
+    private int numberAvailable;
+    @Getter
+    @Setter
+    private String text;
+    @Getter
+    @Setter
+    private LocalDateTime deadline;
+    @Getter
+    @Setter
+    private boolean buyable;
+    @OneToMany(cascade = CascadeType.MERGE)
+    @Getter
+    Collection<TicketOption> possibleOptions;
```

Comment:
Shouldn't this be a set? Then you can also use sets in the methods in the service.